### ISSUE TITLE
Live tile

### ISSUE THREAD

**Issue by benji2020** [hihain/Unigram#12]
Hi

I am fairly new to using this software, but it would appear that the live tile notification isn't available/working.

Regards
Ben

**Comment by hihain** [comment]
You mean the badge? You can deactivate it, if you are unhappy with the update rate. App-wise it gets updated every time there is a change, but the app has no incluence on the time of UI update. This handles Windows Mobile itself. The only way to update it correctly is by opening the app.

In the settings, you can deactivate the badge alltogether or decide if also muted chats should count to the unread chats counter.

**Comment by benji2020** [comment]
Thanks. When the icon is full/mid size it shows an message or preview, but on the smallest square shape icon it should have a number on it for number of messages? (e.g. Outlook icon unread email shows a number) It never seems to for me. It's not a major issue, just wondering if it was the same for you.

**Comment by hihain** [comment]
Yes, it should look like this: 
![image](https://user-images.githubusercontent.com/1473059/88454560-d4331d00-ce70-11ea-8c45-e56f0a554362.png)

**Comment by benji2020** [comment]
If you long press on the icon, then change the size to the smallest square one, do you still get the 6 in the corner?

**Comment by hihain** [comment]
Yes, that is actually my default size. It may take a bit to update and show a few seconds no badge. 
![image](https://user-images.githubusercontent.com/1473059/88455576-a355e680-ce76-11ea-8335-f293e4e92196.png)
